terraform/aws/hashistack.pkr.hcl

```diff
@@ -0,0 +1,70 @@
+variable "hcp_bucket_name"{
+  default="hashistack"
+}
+
+variable "image_name"{
+  default="hashistack"
+}
+
+data "amazon-ami" "aws_base" {
+  filters = {
+    architecture                       = "x86_64"
+    "block-device-mapping.volume-type" = "gp2"
+    name                               = "ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-*"
```

Comment:
16.04 is only under support for the extended security maintenance from Canonical. We shouldn't be creating new content using such an old base.

Reply:
My approach was to leave the existing packer template, old Ubuntu and all. I made that .hcl by converting the existing .json & didn't get into the issue of changing the underlying build.  I will be happy to do that though.

Reply:
To be honest, this whole stack is poorly maintained and I'd rather tear it out than anything else. If your team is willing to take enough ownership over this to update it for HCP Packer, let's update it to something we'd feel good about people running.